### PR TITLE
[READY-TO-MERGE] Document teamSpaceTemplates query (FUTURE view)

### PR DESCRIPTION
**Pull request by a-kent**
@andrewegan Can you review please?

### CODE REVIEW

**Review by andrewegan — COMMENTED** [review]
Missing some information on the API only being accessible by users with the correct offering permission. Otherwise looks okay

**Comment by a-kent** [comment]
@andrewegan Better?

**Review by andrewegan — COMMENTED** [review]
How about:

> Using the teamSpaceTemplates query, a user with the appropriate permission (determined by the offering to which they are subscribed) can fetch a collection of the templates curated by their team admin.

**Comment by a-kent** [comment]
@miguelestrada @jarusso Can someone review please?

File: guides/V1_get_team_space_templates.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'team-space-templates'
+is: 'future'
+---
+
+# Get Team Space Templates
+
+Some space templates are associated with particular teams. 
+Using the teamSpaceTemplates query, a user with the appropriate permission (determined by the offering to which they are subscribed) can fetch a collection of the templates curated by their team admin.
```
Comment (jarusso) [inline guides/V1_get_team_space_templates.md]: could you bold the teamSpaceTemplates word to make it stand out?

Comment (a-kent) [reply guides/V1_get_team_space_templates.md]: Done.

**Pull request merged**